openpype/modules/sync_server/tray/models.py

```diff
@@ -413,6 +414,23 @@ def get_index(self, id):
                 return index
         return None
 
+    def _convert_date(self, date_value, current_date):
+        """Converts 'date_value' to string.
+
+        Value of date_value might contain date in the future, used for nicely
+        sort queued items next to last downloaded.
+        """
+        try:
+            converted_date = None
+            # ignore date in the future - for sorting only
+            if date_value and date_value < current_date:
+                converted_date = date_value.strftime("%Y%m%dT%H%M%SZ")
```

Comment:
Could this be dangerous for different time zones or a machine that is say an hour behind with its hardware clock?

What sorting are we trying to solve, the one in Qt only? Or in the mongodb queries itself?
I feel like we could solve this simpler/better for either of those.

Reply:
I don't think this is a problem. Especially considering the re-write of the site-sync as a OPv4 addon anyways. Not worth the time now.